src/logic/schedule-parser/schedule.parser.ts

```diff
@@ -43,6 +44,12 @@ export class ScheduleParser implements ScheduleProvider {
       if (r && r.length !== 0 && r[0].length !== 0) {
         const rKey = r[0][0].toLowerCase().trim();
         if (rKey === "dni miesiąca") {
+          if (r.length === 3) {
```

Comment:
I don't really like this solution, 
I am making MR to your branch with suggestion how I would solve this if we want to have all "dni miesiąca", "pracownicy dzienni", "dzieci" in one array

Reply:
Please let me know if you agree or think it will make things more complicated

Reply:
https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/pull/268